components/Mitra/index.vue

```diff
@@ -166,6 +246,91 @@ export default {
         this.query.order_by = 'users.created_at'
         this.query.sort_by = 'desc'
       }
+    },
+    verifyUser (item) {
+      this.dataMitra = item
+      const { name, partner } = item || null
```

Comment:
I wondering the usage of `|| null` here, what is it used for? and what happens if `item` is `false`? @yoslie

Reply:
discussion:

Please be careful when doing object destructuring from `null` or `undefined` because it will throw an Error

![Screen Shot 2022-08-25 at 11 13 32](https://user-images.githubusercontent.com/33661143/186573400-b8be8941-69c8-40b7-b457-5f9740808f39.png)

the safest way I think is to destructuring from an empty Object

ref: https://www.samanthaming.com/tidbits/24-prevent-error-with-default-object-when-destructuring/

Reply:
At first I want to set default value for variable name and partner, but after I do research it can's worked like that for deconstructing variable, It supposed to be
const { name = null, partner = null } = item

But I think I dont need it for now, so I will just remove it thanks @agunghide